CONTRIBUTING.md

```diff
@@ -240,6 +241,10 @@ Please feel free to add to or edit these sections.
 
 ### Formatting tips
 
-We use markdown for the book formatting.
+We use [markdown](https://daringfireball.net/projects/markdown/) for the book's
+formatting.
+
+### Attribution and Citations
 
 - Citations: https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
+- Image attribution: Include credit, copyright notice, and license info for all images
```

Comment:
Should we give an example of how to include that info, or point to example in our text?

Reply:
Good idea. Will update after my conference call. Thanks.